jmoreira/jmoreira.md

```diff
@@ -7,6 +7,6 @@
 ## Slack ID: jmoreira
 ## GitHub ID: [zeira42](https://github.com/zeira42)
 ## About me:
-- Solving software and boulder problems
+- Whether it's code or boulders, I’m all about cracking tough problems.
```

Comment:
https://youtu.be/NP4lrVIpbvo?si=cTLlsQb1pQ7mmOYk

Reply:
thats me right before https://www.instagram.com/p/DANn9alykUH/